app/jobs/onceoff/clean_up_user_export_topics.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module Jobs
+  class CleanUpUserExportTopics < Jobs::Onceoff
+    def execute_onceoff(args)
+      slug = "%-#{Slug.for(I18n.t('system_messages.csv_export_succeeded.subject_template').gsub('[%{export_title}]', ''))}"
```

Comment:
I think we can't take the server's locale. If I am not wrong, the user export should be generated based on the `user's` locale?

Reply:
I verified locally that user export messages are generated based on server locale, not user's.

Reply:
What if the server's locale has been changed? If we don't want to loop through all the available locales, we can look through the site settings history. 

Also, it is a bug that the system message isn't generated using the user's locale.

Reply:
We will now loop through all available locales.